Plugins/FormatSwift/Plugin.swift

```diff
@@ -95,8 +95,9 @@ extension AirbnbSwiftFormatPlugin: CommandPlugin {
     // When running on a SPM package we infer the minimum Swift version from the
     // `swift-tools-version` in `Package.swift` by default if the user doesn't
     // specify one manually
```

Comment:
Putting here so we have a thread...

>A project can also have multiple Package.swifts with different tools versions. This is required if a package wants to make use of functionality that is only available in later tools versions, without dropping support for later tool versions. 

For the last sentence, do you mean this? (change in italics)

> This is required if a package wants to make use of functionality that is only available in later tools versions, without dropping support for _earlier_ tool versions. 

I want to make sure that I'm following.

Reply:
Ah yes thank you, that was a typo.